src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNodeKind.cs

```diff
@@ -0,0 +1,182 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Threading;
+
+namespace System.Text.RegularExpressions
+{
+    /// <summary>Specifies the kind of a <see cref="RegexNode"/>.</summary>
+    internal enum RegexNodeKind
+    {
+        // The following are leaves (no children) and correspond to primitive operations in the regular expression.
+
+        /// <summary>A specific character, e.g. `a`.</summary>
+        /// <remarks>The character is specified in <see cref="RegexNode.Ch"/>.</remarks>
+        One = RegexCode.One,
```

Comment:
`RegexCode.One` etc are still magic consts right -- any reason we shouldn't make those an enum too?

Reply:
> RegexCode.One etc are still magic consts right -- any reason we shouldn't make those an enum too?

No, that's just separate :)

Reply:
Ha, I didn't mean you should, I just meant it's a good idea